modules/rodoo-enable-override.adoc

```diff
@@ -40,16 +40,17 @@ metadata:
   namespace: <namespace>                 <1>
 spec:
   restartPolicy: Never                   <2>
+  securityContext:
+    runAsNonRoot: true
+    seccompProfile:
+      type: RuntimeDefault
   containers:
     - name: busybox
+    # TODO: This already had securityContext here on the container, but it was a mix. Updated, does this look okay?
```

Comment:
@s-urbaniak @stlaz I updated this one to what we've been adding. Does this look okay?

Reply:
the changes to this manifest look good to me